Files/Views/ModernShellPage.xaml.cs

```diff
@@ -322,24 +323,32 @@ private void ModernShellPage_QuerySubmitted(ISearchBox sender, SearchBoxQuerySub
             }
         }
 
+        FolderSearch previousSearchInstance;
```

Comment:
Is this needed?

Reply:
yes the search instance is saved for easily refreshing search results by calling SearchAsync again with the same FolderSearch instance

Reply:
Soyry just seen the reference in `Refresh_Click` 👍